From 7b4a712b3ad90fb39c075b63207c14e64db65506 Mon Sep 17 00:00:00 2001
From: Hugh Guan <hhguan@codeaurora.org>
Date: Mon, 12 Sep 2016 17:27:07 +0800
Subject: [PATCH 2/2] default.pa: Load acdb and codec-control modules

Automatically load acdb and codec-control modules for audio
calibration database loader and 3rd-party codec control support
of automotive platform.
Upstream-Status: Inappropriate [oe specific]
Signed-off-by: Hugh Guan <hhguan@codeaurora.org>
---
 src/daemon/default.pa.in | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/src/daemon/default.pa.in b/src/daemon/default.pa.in
index 30bb77c..05c0f27 100755
--- a/src/daemon/default.pa.in
+++ b/src/daemon/default.pa.in
@@ -74,6 +74,14 @@ load-module module-devd-detect
 .endif
 ])dnl
 
+### Automatically load acdb and codec-control modules by QC
+.ifexists module-acdb@PA_SOEXT@
+load-module module-acdb
+.endif
+.ifexists module-codec-control@PA_SOEXT@
+load-module module-codec-control
+.endif
+
 ### Automatically connect sink and source if JACK server is present
 .ifexists module-jackdbus-detect@PA_SOEXT@
 .nofail
-- 
2.17.1

